chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -2162,7 +2153,7 @@ async def get_coin_records(coin_ids: Collection[bytes32]) -> list[CoinRecord]:
         result = await add_spendbundle(mempool_manager, bundle, bundle_name)
         print(result)
         if expected is not None:
-            assert result == (None, MempoolInclusionStatus.FAILED, expected)
+            assert result == (0, MempoolInclusionStatus.FAILED, expected)
```

Comment:
This is unrelated to the patch (addressed in the addendum), it's a result of altering `add_spendbundle`'s failure path.